packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -539,10 +536,15 @@ export interface IPendingMessagesState {
 }
 
 /**
- * Represents the context for the data store like objects. It is used by the data store runtime to
- * get information and call functionality to its parent.
+ * Context for an {@link IDataStore}.
```

Comment:
I don't think calling this a context for data store or IDataStore is correct. This was deliberate named as such to distinguish it from a data store context. The one place where this is not for a data store is container runtime which is an `IFluidParentContext` and is passed to the `ChannelCollection`.

Reply:
I think I misinterpreted "This layout is temporary, as {@link IFluidParentContext} and {@link IFluidDataStoreContext} will converge."


Since this is alpha+legacy and ChannelCollection is internal, we should document it in terms of ChannelCollection.

I have re-split the docs for IFluidDataStoreContext and IFluidParentContext.